config/config/sc-config.yaml

```diff
@@ -192,6 +261,20 @@ harbor:
         requests:
           memory: 32Mi
           cpu: 10m
+      affinity:
```

Comment:
not sure if necessary as it will always run with 1 replica

Reply:
Sure, but it doesn't hurt and then if you scale up to 2 replicas you already have the spreading already in the defaults.

Reply:
yes it doesn't hurt, but from what I can see scaling for internal database and redis is not supported by the chart